101-azure-search-create/azuredeploy.json

```diff
@@ -18,10 +18,12 @@
         "basic",
         "standard",
         "standard2",
-        "standard3"
+        "standard3",
+        "storage_optimized_l1",
+        "storage_optimized_l2"
       ],
       "metadata": {
-        "description": "The SKU of the search service you want to create. E.g. free or standard"
+        "description": "The pricing tier of the search service you want to create. You can have one free service per subscription."
```

Comment:
Why are we explicitly stating "You can have one free service per subscription."?

Reply:
It's the only tier that has this constraint so I thought we should mention it.  But on second thought, who is going to use a template to roll out a free tier?  No one.  I'll change it back to mentioning a few tier examples (basic or standard).

Reply:
It's fine to lead this in, especially if we're trying to clear up customer confusion around quota for free SKUs. I just don't have any context of why this was added.